src/js/common/schemaform/helpers.js

```diff
@@ -129,6 +129,10 @@ export function createRoutes(formConfig) {
     {
       path: 'confirmation',
       component: formConfig.confirmation
+    },
+    {
+      path: '*',
+      onEnter: (nextState, replace) => replace('/')
```

Comment:
I think we would want to use `urlPrefix` here: `replace(formConfig.urlPrefix || '/')`.